src/components/atomic/Checkbox.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { InputHTMLAttributes } from "react";
+import classNames from "classnames";
+
+type CheckboxProps = InputHTMLAttributes<HTMLInputElement> & {
+  labelText?: string;
+};
+
+const Checkbox = ({ className = "", ...props }: CheckboxProps) => {
+  const wrapperStyles = classNames({
+    ["h-5 w-5 relative"]: true,
+  });
+  const checkboxStyles = classNames({
+    ["h-5 w-5 border-2 border-primary rounded appearance-none \
+      checked:bg-primary checked:fill-white -z-1 absolute"]:
```

Comment:
Needs:
- cursor-pointer
- hover and active styles (whatever you feel makes sense, design can review it later)

Reply:
made hover thicken the border width and active change border color to dark blue